FlashDevelop/MainForm.cs

```diff
@@ -1700,6 +1701,9 @@ protected override bool ProcessCmdKey(ref Message msg, Keys keyData)
              */
             else if (ShortcutKeysManager.IsValidExtendedShortcutFirst(keyData) && !appSettings.DisableExtendedShortcutKeys)
             {
+                if (ShortcutManager.AllShortcuts.FirstOrDefault(s => s.First == keyData) == ShortcutKeys.None)
+                    return false;
+
```

Comment:
If this is going to be used, it should be added in line with the if condition above it, like
```c#
else if (ShortcutKeysManager.IsValidExtendedShortcutFirst(keyData)
    && !appSettings.DisableExtendedShortcutKeys
    && ShortcutManager.AllShortcuts.FirstOrDefault(s => s.First == keyData).IsExtended)
```
so that the undefined shortcut would go to the `else if` below.

But this behaviour rather makes it complicated for the users. Regardless of whether shortcut is defined or not, when an undefined shortcut is pressed, the app setting is not set to disable extended keys, and the pressed key is not F1~F24 (contains Ctrl, Alt and/or Shift), is should always be regarded as the first part of the combination.